### ISSUE TITLE
qemu-kvm >= 1.4.0 shouldn't be started with both -nographic and -daemonize

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#445]
Originally reported of Google Code with ID 389.

```
What software version are you running? Please provide the output of "gnt-
cluster --version" and "gnt-cluster version".

gnt-cluster (ganeti v2.7.0beta1) 2.7.0~beta1

<b>What distribution are you using?</b>

Gentoo

<b>What steps will reproduce the problem?</b>
1. Use a kvm version where this patch has been applied: https://lists.gnu.org/archive/html/qemu-devel/2012-10/msg04798.html. I have confirmed this is the case with the 1.4.0 ebuild of qemu on gentoo
2. Create new instance with vnc_bind_address set to default
3. Start the instance

<b>What is the expected output? What do you see instead?</b>
Instance should start. Instead I get the following error: 
Could not start instance: Hypervisor error: Failed to start instance: exited with exit code 1 (-nographic can not be used with -daemonize)

<b>Please provide any additional information below.</b>

One can workaround the issue by configuring vnc_bind_address, which assumingly disables the "-nographic" argument when starting qemu-kvm
```
Originally added on 2013-02-26 17:09:11 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-04-30 17:28:02 +0000 UTC.

Added Labels: Priority-Medium
Added to Milestone: Release2.8

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-05-01 11:19:25 +0000 UTC.

Changed State: Accepted
Added to Milestone: Release2.7
Removed from Milestone: Release2.8

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 99c7cd5be025e86745aa46003ca0962609e0b4e2
Author: Guido Trotter <ultrotter@google.com>
Date:   Wed May 1 11:37:54 2013 +0200

    kvm: Use -display none rather than -nographic
    
    Newer versions of kvm (from 1.4) don't support -nographic with
    -daemonize. When -display exists, use -display none instead.
    
    This fixes Issue 389.
    
    Signed-off-by: Guido Trotter <ultrotter@google.com>
    Reviewed-by: Thomas Thrainer <thomasth@google.com>

```
Originally added on 2013-05-02 08:46:04 +0000 UTC.

Mentioned Issue with GoogleCode ID 389 has been migrated to GitHub with ID #445.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
